customcontext/texture/drawhelper.h

```diff
@@ -0,0 +1,59 @@
+/****************************************************************************
+**
+** Copyright (C) 2015 Christophe Chapuis, author: <chris.chapuis@gmail.com>
```

Comment:
You just moved the existing code, so I'd say keep the existing copyright from there.